scripts/publish.sh

```diff
@@ -53,20 +64,28 @@ ARTIFACT_FILE="$(ci-pkginfo -t pkgname)-$(ci-pkginfo -t pkgsemver).tgz"
 DEPLOY_VERSION="$([[ ${ARTIFACT_FILE} =~ vuepress-site-(.*)\.tgz ]] && echo ${BASH_REMATCH[1]})"
 ARTIFACT_PATH="@okta/vuepress-site/-/${ARTIFACT_FILE}"
 
-if ! trigger_and_wait_release_promotion_task 60; then
-  echo "Automatic promotion failed..."
-  exit ${BUILD_FAILURE}
+# Only auto-promote to npm-release on main branch
+if [[ $BRANCH == "master" ]]; then
+  if ! trigger_and_wait_release_promotion_task 60; then
+    echo "Automatic promotion failed..."
+    exit ${BUILD_FAILURE}
+  fi
+  # update target registry to release for production deploy
+  REGISTRY_REPO="npm-release"
 fi
 
-if ! send_promotion_message "${DEPLOY_ENVIRONMENT}" "${ARTIFACT_PATH}" "${DEPLOY_VERSION}"; then
+# tell conductor to deploy
+if ! send_promotion_message "${DEPLOY_ENVIRONMENT}" "${REGISTRY_REPO}" "${ARTIFACT_PATH}" "${DEPLOY_VERSION}"; then
```

Comment:
`REGISTRY_REPO` is the target artifact repo. When this runs against `master`, `REGISTRY_REPO` will be `npm-release`. When it's running on `staging`, `REGISTRY_REPO` will be `npm-topic`.

The `DEPLOY_ENVIRONMENT` parameter sets which conductor deployment pipeline (`dev` vs `prod`) will be triggered